conformance_tests/sysman/test_sysman_events/src/test_sysman_events.cpp

```diff
@@ -453,7 +453,14 @@ TEST_F(
              << _ze_result_t(ZE_RESULT_ERROR_UNSUPPORTED_FEATURE);
     }
     for (auto power_handle : power_handles) {
-      auto energy_threshold = lzt::get_power_energy_threshold(power_handle);
+      zes_energy_threshold_t energy_threshold = {};
+      auto status =
+          lzt::get_power_energy_threshold(power_handle, &energy_threshold);
+      if (status == ZE_RESULT_ERROR_UNSUPPORTED_FEATURE) {
+        continue;
+      } else {
```

Comment:
minor : else is not needed, since we are continuing

Reply:
We want to fail on any other return codes other than UNSUPPORTED, hence checking if its equal to SUCCESS.

Reply:
> We want to fail on any other return codes other than UNSUPPORTED, hence checking if its equal to SUCCESS.

Checking for success is needed.
The "else" is not needed and the code can fall through